dev-libs/libpreludedb/files/libpreludedb-5.1.0-fix_py38.patch

```diff
@@ -0,0 +1,11 @@
+--- a/bindings/python/_preludedb.cxx	2019-07-13 08:39:59.744285379 +0200
++++ b/bindings/python/_preludedb.cxx	2019-07-13 08:41:47.364117843 +0200
```

Comment:
You don't really need the timestamps here, it doesn't add any value. Please remove them.

Reply:
done